README.md

```diff
@@ -3,21 +3,19 @@ Code Runner: A tool to help run your assignments
 Code Runner is a web application to run code pertaining to your assignment using user provided configurations.
 
 ## Features
-There are two roles in the application - Professor and Student.
-#### Professor
-- Professor provides course specific configurations for creating suitable docker environment.
-- Course specific configurations include specifying operating system and its version, programming language and compiler.
+There is one role in the application - Student.
 #### Student
 - Students can submit their assignments and view the output through a web page.
-- They can also enter the optional command line arguments that are required to run their assignment.
+- They can enter the assignment working directory, commands to compile and run the assignment.
+- They can also enter the optional command line arguments required to run their assignment.
```

Comment:
instead list all the things that can be provided by the user and explain each of them alongside their listing.

Reply:
Okay